src/stream/sink/udp_sink.rs

```diff
@@ -142,7 +182,7 @@ impl UdpSink {
             })
             .collect::<Vec<String>>()
             .join(",");
-        let description = format!("multiudpsink clients={addresses}");
```

Comment:
try to avoid creating unnecessary noise in commits that are unrelated to the functional change.
you can do it in individual commits.

Reply:
This is related: because the property self.addresses was added, the meaning of `addresses` would be confusing in the context of this function.

Reply:
Oh, I see it now, thanks!